### DISCUSSION TITLE
CWOps CW Open Contest

### DISCUSSION THREAD

**Discussion by n4eel** [w7sst/MorseRunner#220] [New Contest Suggestions]
Hello! I was hoping to get some practice before this weekend's annual CW Open contest, but couldn't find a contest with the exchange configuration of " QSO#, NAME " - anything with a serial number is " RST, # ", anything with a name is " RST, NAME " or "NAME, ST". So I'm hereby submitting a request for the CWOps CW Open! here's a link to the contest rules: [CW OPEN](https://cwops.org/cwops-tests/cw-open/#team)

thanks a kazillion, 
N4EEL

**Comment by w7sst** [comment]
Hi @n4eel,
Thank you for submitting this contest request. We will add it to the list.

Thank you and 73,
Mike W7SST

**Comment by w7sst** [comment]
Hi @n4eel,
I was starting to code up this contest and had a couple of questions...

1. Is the serial number sent with cut numbers (e.g. `T` for zero, `N` for 9, etc.)?
2. The exchange is `<serial> <name>`. Example: `123 WIGG`.
3. Do you ever send, or have your heard others send, an `A` as a cut version of the number one?

Thank you,
Mike W7SST
